src/tenant/admin.py

```diff
@@ -130,6 +170,106 @@ def get_queryset(self, request):
                     tenant.update_cached_fields()
         return qs
 
+    def delete_queryset(self, request, queryset):
+        """
+        Given a queryset, delete (but leave the schema) it from the database.
+        """
+        # for reference: https://django-tenants.readthedocs.io/en/stable/use.html#deleting-a-tenant
+        for obj in queryset:
+            obj.delete(force_drop=False)  # delete model, but *DO NOT* drop schema
```

Comment:
This seems like a criticla line to be covered by tests, but codecov patch note says it is missed.  However, what is this method for?  Didn't queryset deletion (deletion from list view) get removed? Or am I confused?